server/Dockerfile

```diff
@@ -9,17 +9,23 @@ USER root
 
 RUN yum install -y epel-release && yum install -y jq && yum clean all
 
-USER jboss
-
 RUN cd /opt/jboss/ && curl -L https://downloads.jboss.org/keycloak/$KEYCLOAK_VERSION/keycloak-$KEYCLOAK_VERSION.tar.gz | tar zx && mv /opt/jboss/keycloak-$KEYCLOAK_VERSION /opt/jboss/keycloak
 
 ADD docker-entrypoint.sh /opt/jboss/
 
 ADD setLogLevel.xsl /opt/jboss/keycloak/
 RUN java -jar /usr/share/java/saxon.jar -s:/opt/jboss/keycloak/standalone/configuration/standalone.xml -xsl:/opt/jboss/keycloak/setLogLevel.xsl -o:/opt/jboss/keycloak/standalone/configuration/standalone.xml
+RUN sed -i 's/\(logger.handlers=CONSOLE\),FILE/\1/' keycloak/standalone/configuration/logging.properties
 
 ENV JBOSS_HOME /opt/jboss/keycloak
 
+RUN mkdir $JBOSS_HOME/standalone/{data,log} && \
```

Comment:
This block sets up necessary writable dirs to be able to run as arbitrary user. In an OpenShift deployment, the run as user is set to arbitrary uid based on the project uid range. The `chmod`s below set up stuff as required to be readable/writable by anyone in root group (not a security risk btw). See https://docs.openshift.com/enterprise/3.2/creating_images/guidelines.html#use-uid for more details.